NEWS.md

```diff
@@ -1,3 +1,23 @@
+## 19.3.0 2020-07-28
+
+* Add'Agedtolost'statustoalloweditemstatuseslist (MODINVSTOR-503)
+* MakemigratePrecedingSucceedingTitles.sqlidempotent (MODINVSTOR-541)
+* Separatefilteringfromresponsegenerationinoai-pmhview (MODINVSTOR-536)
+* Itemrecord.Effectivecallnumber(item),eyereadable.SearchonexactdatainCallnumberdataelement (MODINVSTOR-481)
+* Holdingsrecord.Callnumber,eyereadable.SearchonexactdatainCallnumberdataelement (MODINVSTOR-480)
+* Searchquerieswithoutdatabaseindex (MODINVSTOR-472)
+* Upgradetoraml-module-builder(RMB)30.2.4 (MODINVSTOR-532)
+* UpdatetoRMBv30.2.3 (MODINVSTOR-531)
+* Expandoai-pmhviewwithadditionalfields (MODINVSTOR-498)
+* OAI-PMHviewdoesn'trespondinBugFest (MODINVSTOR-527)
+* UpgradetoRMB30.2.2 (MODINVSTOR-528)
+* Changepermissionslocationforoai-pmhview (MODINVSTOR-524)
+* AddIndexforInstancesFull-textSubjectsSearch (MODINVSTOR-499)
+* UpgradeRMBtov30.2.0 (MODINVSTOR-522)
+* Postgresrequiresspecialpermissionstodisabletriggersformigrations (MODINVSTOR-476)
+* UpgradeRMBto30.0.3,fixinginconsistenthitcounts(totalRecords)estimation (MODINVSTOR-519)
+* BulkdownloadInstanceUUIDsOOM (MODINVSTOR-465)
```

Comment:
Why are there no spaces in these notes?

From my perspective, these notes are intended to convey to someone using the module what significant things have changed in this release?

Typically, I also include information about interfaces that have changed - see https://github.com/folio-org/mod-inventory-storage/releases/tag/v19.2.0 as an example.

I don't think there is any need to have multiple lines for multiple upgrades to RAML Module Builder, these can be consolidated to a single line referring to the latest version upgraded to.

Reply:
Done.